database/migrations/2022_03_18_125356_add_auto_start_duration_to_multiplayer_rooms.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddAutoStartDurationToMultiplayerRooms extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('multiplayer_rooms', function (Blueprint $table) {
```

Comment:
Hesitant to keep adding columns for each setting like this (we've moved to `json` storage for other similar cases). Fine this time to keep the changes simple, but just noting my concerns.

Reply:
Absolutely. And this column is being used for both playlists and multiplayer which have different requirements that are being bunched up together here.

That sort of change is a little too involved for me to make in quick order so I'll let someone else handle the web-side of things whenever that happens.